app/assets/javascripts/models/beta/iteration.js

```diff
@@ -1,21 +1,154 @@
-import moment from 'moment';
+import moment from "moment";
+import * as Story from "./story";
 
 const weeksBetween = (dateA, dateB) =>
-  moment(new Date(dateA)).diff(moment(new Date(dateB)), 'week');
+  moment(new Date(dateA)).diff(moment(new Date(dateB)), "week");
 
 const getIterationForDate = (date, project) => {
   const weeks = weeksBetween(date, project.startDate);
   const iterationQuantity = Math.ceil(weeks / project.iterationLength);
 
-  return project.iterationLength > weeks ? iterationQuantity : iterationQuantity + 1;
+  return project.iterationLength > weeks
+    ? iterationQuantity
+    : iterationQuantity + 1;
 };
 
-export const getCurrentIteration = (project) => (
-  getIterationForDate(new Date(), project)
-);
+export const getDateForIterationNumber = (iterationNumber, project) => {
+  return moment(new Date(project.startDate))
+    .startOf("isoWeek")
+    .add(iterationNumber, "weeks")
+    .format("ddd MMM Do Y");
+};
+
+export const getCurrentIteration = project =>
+  getIterationForDate(new Date(), project);
 
 export const getIterationForStory = (story, project) => {
-  if (story.state === 'accepted') {
+  if (story.state === "accepted") {
     return getIterationForDate(new Date(story.acceptedAt), project);
   }
-}
+};
+
+const createSprint = (sprintNumber = 0, startDate = 0, isFiller = false, velocity) => ({
+  number: sprintNumber + 1,
+  startDate: startDate,
+  points: 0,
+  completedPoints: null,
+  stories: [],
+  isFiller: isFiller,
+  remainingPoints: velocity 
+});
+
+const createFillerSprints = (size, initialNumber, project) => {
+  let fillerSprints = [];
+  for (let i = 0; i < size; i++) {
```

Comment:
I think we can use [_.times](https://lodash.com/docs/#times) here to return the array of filler sprints